src/lib/import_export/binary/binary_writer.hpp

```diff
@@ -193,21 +208,25 @@ class BinaryWriter {
    * NULL values¹                | vector<bool> (BoolAsByteType)       | Size * 1
    * Dictionary size             | uint32_t                            | 4
    * Dictionary                  | vector<char>                        | Dictionary size * 1
-   * string offset size          | uint32_t                            | 4
-   * string offset data size²    | uint32_t                            | 4
-   * string offset²              | uint32_t                            | size * 4
+   * String offset size          | uint32_t                            | 4
+   * Vector compress. bit width²³| uint8_t                             | 1
+   * String offset values²³      | uint8_t                             | Rows * (vector compr. bit width) / 8
+   *                                                                     rounded up to next multiple of word (8 byte)
+   * String offset values²⁴      | uint(8|16|32)_t                     | Rows * width of offset vector
```

Comment:
This is never the case, right?

Reply:
I was a bit too eager there. Removed it.